client/locations-list-map.js

```diff
@@ -1247,12 +1250,17 @@ function refreshList(data, filters) {
   }
 }
 
-function onFilterChange(data, prefix, idx, selected, filters) {
+function onFilterChange(data, prefix, idx, selected, filters, selectr) {
   const primaryFilter = filters[prefix] && filters[prefix][Object.keys(filters[prefix])[idx]];
   if (!primaryFilter) {
     return;
   }
-
+  const temp = filters[prefix][Object.keys(filters[prefix])[idx]].name;
```

Comment:
nit: can we have a more descriptive variable name? I'm not 100% certain what the semantics of temp are...